skate-plugin/src/main/kotlin/com/slack/sgp/intellij/SkatePluginSettings.kt

```diff
@@ -55,10 +55,31 @@ class SkatePluginSettings : SimplePersistentStateComponent<SkatePluginSettings.S
       state.translatorFileNameSuffix = value
     }
 
+  var isLinkifiedFeatureFlagsEnabled: Boolean
```

Comment:
Can you add these to the settings UI and do error handling for missing urls/annotations there?